scripts/pi-hole/js/groups-domains.js

```diff
@@ -503,16 +503,16 @@ function addDomain() {
     commentEl = $("#new_regex_comment");
   }
 
-  const comment = utils.escapeHtml(commentEl.val());
+  const comment = commentEl.val();
```

Comment:
No need to escape the comment, otherwise it will end up escaped in the database.

Reply:
We should also remove the coment escaping for clients, adlists, groups....